utils/run_test.sh

```diff
@@ -98,9 +98,10 @@ if [ -d "/mnt/daos" ]; then
     run_test "${SL_PREFIX}/bin/vea_ut"
     run_test src/rdb/raft_tests/raft_tests.py
     # Satisfy CGO requirements for go-spdk binding and internal daos imports
-    LD_LIBRARY_PATH="${SL_PREFIX}/lib:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
+    LD_LIBRARY_PATH=
+    "${SL_PREFIX}/lib64:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
```

Comment:
You can't put these on a separate line, at least not without a `\` at the end of the first line.  I.e change the above two lines to:

```suggestion
    LD_LIBRARY_PATH=\
    "${SL_PREFIX}/lib64:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
```